### PR TITLE
Dart 3 breaking change entry

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: src/release/breaking-changes/index.md [hunk 1]
```diff
@@ -30,12 +30,14 @@ release, and listed in alphabetical order:
...
 ### Not yet released to stable
 
+* [Dart 3 changes in Flutter v3.10 and later][]
 * [Deprecated API removed after v3.7][]
 * [Removed `ignoringSemantics`][]
 * [The window singleton is deprecated][]
 * [Migration guide for `RouteInformation.location`][]
 * [At least one clipboard data variant must be provided][]
 
+[Dart 3 changes in Flutter v3.10 and later]: {{site.url}}/release/breaking-changes/dart3
```
Comment (parlough) [inline src/release/breaking-changes/index.md:40]: Are you going to introduce a page on the Flutter site too or do you mean to link this to the Dart page in the PR mentioned?

```suggestion
[Dart 3 changes in Flutter v3.10 and later]: {{site.dart-site}}/resources/dart-3-migration
```

Comment (mit-mit) [reply src/release/breaking-changes/index.md:40]: I meant to link to Dart; thanks for catching!

**Comment by mit-mit** [comment]
Destination link is live: https://dart.dev/resources/dart-3-migration

**Comment by mit-mit** [comment]
Yup, no problem. @khanhnwin would you mind merging this one on my behalf when ready?

**Review by parlough — APPROVED** [review]

**Pull request merged**